README.md

```diff
@@ -67,4 +67,10 @@ To run the unit test please use command poetry run pytest
 
 login with your control IP ssh ec2-user@
 
-run your playbook with "ansible-playbook Playbook.yaml -i Inventory.ini"
\ No newline at end of file
+run your playbook with "ansible-playbook Playbook.yaml -i Inventory.ini"
+
+## How to start dockerfiles
+run these commands 
+docker build --tag todo-app .
+docker run --env-file .env todo-app
+docker run --entrypoint bash -it todo-app ( Run to look into container and debug )
```

Comment:
This is a good start, but it doesn't actually capture how you expect users to prepare & run their images as far as I can tell!
Since you moved (correctly) to the multi-stage Dockerfile, this is no longer valid. Instead you will want to list out the prod & dev instructions separately:

```
## How to start dockerfiles
To build the production container:
`docker build --tag todo-app:prod --target production .`
And to build the dev container:
`docker build --tag todo-app:dev --target development .`

You can then run both using `docker compose up`

You can also debug the containers with:
`docker run --entrypoint bash -it todo-app:<env>`
```

Alternatively, if you adjust the `docker-compose.yml` as suggested below then you might be able to get away with just offering

```
## How to start dockerfiles
You can build & run both dev and production containers using `docker compose up --build`

You can also debug the containers with:
`docker run --entrypoint bash -it todo-app:<env>`
```
```